language/ast.py

```diff
@@ -0,0 +1,779 @@
+from langkit.dsl import ASTNode, Field, abstract  # type: ignore
+
+
+@abstract
+class RFLXNode(ASTNode):
+    """
+    Root node class for the RecordFlux DSL language
+    """
+
+
+class NullID(RFLXNode):
+    """
+    "null" identifier
+    """
+
+
+class UnqualifiedID(NullID):
+    """
+    Simple, unqualified identifiers, i.e. identifiers without a package part (e.g. "Foo")
+    """
+
+    token_node = True
```

Comment:
Is the `null` identifier equal to what is described by the `null` keyword? I wonder how it makes sense that other identifiers are also null identifiers.

Reply:
Added a new base class in 5416315.